src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Contracts/Definition.cs

```diff
@@ -14,5 +14,37 @@ public static readonly
             RequestType<TextDocumentPosition, Location[]> Type =
             RequestType<TextDocumentPosition, Location[]>.Create("textDocument/definition");
     }
+
+    /// <summary>
+    /// Error object for Definition
+    /// </summary>
+    public class DefinitionError
+    {
+        /// <summary>
+        /// Error message 
+        /// </summary>
+        public string message { get; set; }
+    }
+
+    /// <summary>
+    /// Result object for Definition
+    /// </summary>
+    public class DefinitionResult
+    {
+        /// <summary>
+        /// True, if definition error occured
+        /// </summary>
+        public bool Error;
```

Comment:
This should be HasError. Boolean properties should follow standard "IsXyz", "HasXyz", "DidXyz" patterns.